tests/installation/zdup.pm

```diff
@@ -177,6 +106,8 @@ sub run {
     }
 
     assert_screen "zypper-dup-finish";
+    zypper_call "in issue-generator";
+    record_soft_failure 'bsc#1133636';
```

Comment:
Seems like you are not checking the actual error condition for the bug

Reply:
This is a workaround for bsc#1133636, install issue-generator before migration then the bsc#issue-generator won't happen.

Reply:
I think we are still talking about different things here. The `record_soft_failure` will be called regardless if the bug is still present or not which can lead to false-positive reports to the bug. I suggest to check if the bug is present and only if this is the case then call `record_soft_failure`. For example – if I understand the bug correctly:

```perl
if (script_run '! test -e /etc/issue') {
    record_soft_failure 'bsc#1133636';
    zypper_call 'in issue-generator';
}
```

Reply:
Yes, thanks, I misunderstand your meaning. Updated code. Thanks.